pkg/metrics/collector.go

```diff
@@ -96,21 +102,254 @@ func (c *metricsCollector) publishVolumeStats(ctx context.Context, volume *types
 	)
 }
 
+type DriveStats struct {
+	ReadSectors  int64
+	ReadTicks    int64
+	WriteSectors int64
+	WriteTicks   int64
+	TimeInQueue  int64
+}
+
+func (c *metricsCollector) publishDriveStats(ctx context.Context, drive *types.Drive, ch chan<- prometheus.Metric) {
+	device, err := c.getDeviceByFSUUID(drive.Status.FSUUID)
+	if err != nil {
+		klog.ErrorS(
```

Comment:
If the device is not found by its FSUUID, this means the drive is not READY. we should also export this information as a metric.